inc/custom-header.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Sample implementation of the Custom Header feature
+ * http://codex.wordpress.org/Custom_Headers
+ *
+ * You can add an optional custom header image to header.php like so ...
+
+ *
+ * @package twentynineteen
+ */
+
+/**
+ * Set up the WordPress core custom header feature.
+ *
+ * @uses twentynineteen_header_style()
+ */
+function twentynineteen_custom_header_setup() {
+	add_theme_support( 'custom-header', apply_filters( 'twentynineteen_custom_header_args', array(
+		'width'                  => 2000,
+		'height'                 => 280,
+		'flex-height'            => true,
+		'wp-head-callback'       => 'twentynineteen_header_style',
+	) ) );
+}
+add_action( 'after_setup_theme', 'twentynineteen_custom_header_setup' );
+
+if ( ! function_exists( 'twentynineteen_header_style' ) ) :
+	/**
+	 * Styles the header image and text displayed on the blog.
+	 *
+	 * @see twentynineteen_custom_header_setup().
+	 */
+	function twentynineteen_header_style() {
+		$header_text_color = get_header_textcolor();
```

Comment:
I recommend turning off support for the header text color. An overlay is provided to ensure readability. See also the discussions for Twenty Seventeen.

If header text color is a feature, it should be coordinated with the other color customization options instead of being implemented here.